src/test/java/org/codeforamerica/shiba/output/pdf/PdfIntegrationTest.java

```diff
@@ -437,12 +434,7 @@ void shouldAddAuthorizedRepFieldsIfYes() {
             testPage.enter("getMailNotices", YES.getDisplayValue());
             testPage.enter("spendOnYourBehalf", YES.getDisplayValue());
 
-            testPage.enter("helpersFullName", "defaultFirstName defaultLastName");
-            testPage.enter("helpersStreetAddress", "someStreetAddress");
-            testPage.enter("helpersCity", "someCity");
-            testPage.enter("helpersZipCode", "12345");
-            testPage.enter("helpersPhoneNumber", "7234567890");
-            testPage.clickContinue();
+            fillOutHelperInfo();
```

Comment:
Moving helper methods from `UserJourneyPageTest` to `AbstractBasePageTest` enabled us to use some of those helper methods in PdfIntegrationTest. This is one example.